src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+# Revolutionaries Rework
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| SlamBamActionman<br>ScarKy0 | :x: No | TBD |
+
+## Overview
+
+Revolutionaries as a gamemode is a mechanically and thematically interesting concept; sedition against the station's established authority sounds fun and there is a lot of design space to work within to represent this in mechanics. Previous iterations of Revolutionaries (Revs) have however been seen as encouraging too much combat with too snowball-y rounds for either side, frustrating "metagameable" mechanics and design that both directly and indirectly encourages removing a sizeable number of players from the round. 
+
+This rework intends to address many of the critiques of the original Revs gamemode while still keeping to the theme of exaggerated "Space Communism", allowing for the Revs faction to take over the station via stealth, subterfuge, blatant opposition or violence. 
+
+This design document is inspired by ideas presented by the the following works and authors:<br>
+https://github.com/space-wizards/docs/pull/251 - CaelleakNavinski<br>
+https://github.com/space-wizards/docs/pull/146 - illdoodle<br>
+ScarKy0
+
+### Breakdown
+<sup>In this document, the default side players belong to will be referred to as either "crew" or "Command".</sup>
+
+Revolutionaries are a team conversion antag: a small number of players begin as antagonists, but are able to convert crew to antags, with the antag faction's strength largely being dependent on the number of conversion. As a conversion antag, a round with Revolutionaries should attempt to follow the stages below. 
+
+- Initial conversions
+  - This is the stage where the antag begin conversion crew - this should be done stealthily, and failing to do so may result in a swift win for crew. To their benefit the antag is given some kind of initial advantage, providing a quick power bump to get things going.
+- Expansion with risk
+  - During this stage the antag faction grows, exhibiting signs and actions that may be more noticeable by crew. Staying stealthy becomes more difficult due to the number of antags growing, pushing the antags into more direct confrontration.
+- Confrontation with crew
+  - The antag faction has been exposed and crew are now working to counteract the antags. Depending on how well the two previous stages have gone the crew may be in either a good or poor position to fight back.
+- Climax
+  - An encounter between the antags and crew that largely decides the outcome of the round. 
+- Resolution
+  - After the climax the round finishes up, resolving the round for remaining players. Usually this is the last few minutes before the evacuation shuttle arrives.
```

Comment:
It could be interesting if there was a sort of capture the flag aspect to the end of the round. The heads think they have weeded out most of the Revs so they go to the bridge and hit a facility secured button telling centcom that they think the station is secured. This starts a ten minute timer and during this time either command or Revs can capture key areas on the station via some means. At the end of the station secured timer the round ends and it's revealed who has control of the station.

There are some real kinks with this idea like what incentives heads to hit the button, where are the key points on the station that need securing, and what if all heads are killed before that point. This does have the upside of no shuttle and no need for TDM.